docs/design/operator.md

```diff
@@ -85,10 +85,12 @@ metadata:
 spec:
   logLevel: 1
   driverSpecDefaults:
-    logging:
+    log:
       logLevel: 5
-      maxfiles: 5
-      maxLogSize: 10M
+      Periodicity: daily
+      MaxLogSize: 500M 
+      MaxFiles: 5
+      logHostPath: /var/lib/log 
```

Comment:
```suggestion
      periodicity: daily
      maxLogSize: 500M 
      maxFiles: 5
      logHostPath: /var/lib/cephcsi 
```

Reply:
why this is /var/lib/log?  it should be `/var/lib/cephcsi` lets keep it same in all the places